From ef0d9470c853ee8bbef33db4708d3657646ad8f8 Mon Sep 17 00:00:00 2001
From: Tim Orling <ticotimo@gmail.com>
Date: Sun, 27 Feb 2022 15:28:17 -0800
Subject: [PATCH] python3-dtc: fix wheel build

The wheel that is built is ${S}/dist/libfdt-1.6.2.dev39+g4048aed.d20220103-cp310-cp310-linux_x86_64.whl
Set PYPA_WHEEL to match this.

Signed-off-by: Tim Orling <tim.orling@konsulko.com>
---
 recipes-kernel/dtc/python3-dtc_1.6.1.bb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/recipes-kernel/dtc/python3-dtc_1.6.1.bb b/recipes-kernel/dtc/python3-dtc_1.6.1.bb
index 449d071..671d280 100644
--- a/recipes-kernel/dtc/python3-dtc_1.6.1.bb
+++ b/recipes-kernel/dtc/python3-dtc_1.6.1.bb
@@ -19,6 +19,8 @@ SRCREV = "4048aed12b81c5a0154b9af438edc99ec7d2b6a1"
 PV = "1.6.1+git${SRCPV}"
 S = "${WORKDIR}/git"
 
+PYPA_WHEEL = "${S}/dist/libfdt-1.6.2*.whl"
+
 inherit setuptools3 pkgconfig
 
 BBCLASSEXTEND = "native nativesdk"
-- 
2.25.1

